test/tld.js

```diff
@@ -56,7 +79,7 @@ describe('tld.js', function () {
       expect(tld.isValid('.com')).to.be(false);
     });
 
-    it('should be falsy on dotless hostname', function () {
+    it.skip('should be falsy on dotless hostname', function () {
```

Comment:
The tests would not pass otherwise?

Reply:
Actually, a hostname without dots seems to be valid (according to the firefox parser). So `localhost` is a valid hostname, strictly speaking, but we won't find a valid public suffix for it, so it will be rejected. Unless we specify it in `validHosts`. So I disabled these tests, should I remove them altogether?

We can double check if we have enough test cases to cover the behavior of all the functions for such corner cases (`getPublicSuffix`, `getDomain`, etc.) and make sure we offer a consistent behavior.